### DISCUSSION TITLE
Are dual battery cells supported?

### DISCUSSION THREAD

**Discussion by aviatrix** [VR-25/acc#144] [Q&A] [answered]
Hello, I've got an OP 9 Pro, which has dual cell batteries. I am concerned/cautious about installing ACC on my phone because i'm unsure if ACC has support for dual cell batteries. 
Any insights/tips/info on this?

**Comment by VR-25** [comment, marked as answer]
It does support multi cell batteries.
However, it'll not recognize voltage higher than 4.4V (yet to be implemented) - meaning, if one wants to control the peak charging voltage of a battery pack in series, they'll have to specify the control files, as well as default voltage values to use.
To reiterate, everything will work out of the box, except automatic voltage control files detection.
